### ISSUE TITLE
multi terminal 

### ISSUE THREAD

**Issue by YouSour** [Swordfish90/cool-retro-term#247]
hi , i'm new for cool-retro-term , cool-retro-term can open multi terminal or not ? if not i hope have this feature soon . thank advance !

**Comment by ppamorim** [comment]
+1

**Comment by VPhantom** [comment]
Just a thought, but given the retro philosophy of CRT, people looking for multiple tabs should look to using the "screen" command as we did back when we used those monitors. :)

**Comment by YouSour** [comment]
:+1:

**Comment by a-andreyev** [comment]
My two cents: if you do not want to mess with the GNU Screen, I guess the task to open several windows fits more for your WM/DE.

**Comment by phroun** [comment]
I think if this had the edge of a television style knob showing on the skin (at the upper right edge), which switched between "VHF" channels 2-13 and off, and when flipping, the horizontal/vertical sync spazzed out just a little, this would be pretty awesome.  Channel 3 should be where the first terminal shows up since that's where most devices frequency was set.  Empty channels could show different kinds of static...   Caveat:  If this makes the skin wider, it needs to be an option not an always-on thing.
![knob](https://cloud.githubusercontent.com/assets/13051806/14393557/6c1af4dc-fd7c-11e5-9173-eeedc5d2283d.jpg)

**Comment by dessalines** [comment]
I want to use this as my main driver, please add tabs!

**Comment by josefnpat** [comment]
Also look into `tmux`. Works with this software like a charm.

**Comment by larsbrinkhoff** [comment]
I agree with @vphantom.  This would take away from the retro experience.

**Comment by adammathes** [comment]
For those that are interested in multiple cool-retro-terms on their desktop, I've started a branch.

https://github.com/adammathes/cool-retro-term/tree/multiterm

It works reasonably well on MacOS for me so far but I have not tested what happens on *nix with these changes.

And also I have never written any QML/QT before so feedback is welcomed.

**Comment by enkeli** [comment]
I disagree with @larsbrinkhoff. If implemented like @phroun suggests, it would be awesome and full retro!
I've been using screen (i use it along with byobu (also tmux) a lot in other terminals) but there are some issues with some programs that outputs formatted text, and that is the only thing keeping me from using CRT as my main terminal.

**Comment by okasion** [comment]
For those who would like to use this terminal as a main one (I know, it's just eye-candy, but it is of the best kind), why wouldn't it be possible to have a terminal option?

**Comment by okasion** [comment]
Also, how difficult could be code shift+insert as paste command?

**Comment by dessalines** [comment]
`control+shift+v` is the standard terminal paste.

**Comment by okasion** [comment]
That works too... thanks.

**Comment by bconfortin** [comment]
I would love CRT to support tabs.
It is so pretty and looks a lot like Fallout's terminals.
With the tabs feature, It would definitely turn into my daily driver terminal.

**Comment by atmadeep** [comment]
So are we getting a multi terminal feature or not?

**Comment by heeplr** [comment]
I agree that tabs are not in the scope of this project and that screen/tmux/... is the solution to that.

But when using screen, I find that the default key combo "Ctrl+a Backspace" doesn't work to move to the previous session. "Ctrl+a n" to switch to the next session works fine.

Can we have Ctrl+a Backspace?

**Comment by allisonober** [comment]
I came up with a limited, but very simple way for multiple terminal windows. In MacOS, I copied the application and renamed it to make it a separate app. Now I have two windows to use, which act like separate applications.

<img width="1096" alt="Screen Shot 2020-10-16 at 10 35 33 AM" src="https://user-images.githubusercontent.com/52665928/96272935-c1764300-0f9c-11eb-8b21-330b0835a0c7.png">

The downsides are configs won't be shared (you'll have to export in one and import in the other to update your profiles), and you'll need to repeat this depending on how many extra windows you want.

The up side is this is a very easy technique that can likely work on any OS. And a bonus, you can now cmd+tab between the terminals, which I actually prefer.

It's a bit silly, and you don't get keybindings, but it's simple and works. Two or three terminals is all I usually need, so this works well for me.

**Comment by sgnls** [comment]
As per [#555](https://github.com/Swordfish90/cool-retro-term/issues/555), I am opening multiple instances absolutely fine via a keyboard shortcut... each trigger fires a new instance.

That aside, contending with the issue you're noting about importing the profile, you should look to use the flag to specify this for each instance;

```
$ cool-retro-term -p Futuristic
$ cool-retro-term -p Default
```

Unless I'm missing something, if you were to create a shortcut (keyboard or old-school), that should suffice without having to duplicate the application. If you open one, you can open another too;

`$ cool-retro-term &`

**Comment by frnco** [comment]
> Just a thought, but given the retro philosophy of CRT, people looking for multiple tabs should look to using the "screen" command as we did back when we used those monitors. :)

Or emacs, you can do multiple buffers plus split the screen plus pretty much anything...

**Comment by SamuelBanya** [comment]
@frnco 
Please tell me how you are running Emacs on 'cool-retro-term' as there is no way to use the 'Command' key on MacOS at all in 'cool-retro-term'.

**Comment by Ganton** [comment]
> @okasion Also, how difficult could be code shift+insert as paste command?
There's a pull request: https://github.com/Swordfish90/cool-retro-term/pull/702

At least we can add a "thumbs up" to that pull request.

**Comment by threefcata** [comment]
My two cents. On recent Mac OS that's got the 'Shortcuts' app. You can simply go to this app and create a shortcut that 'runs a shell script' like this. Don't forget to assign a keyboard shortcut on the detail pane on the right.

![image](https://github.com/user-attachments/assets/5af61bf5-ea38-4a9e-96ea-21c31324be9a)
